bulbs/recirc/mixins.py

```diff
@@ -41,13 +41,25 @@ def save(self, *args, **kwargs):
     def get_recirc_content(self, published=True, count=3):
         """gets the first 3 content objects in the `included_ids`
         """
+        query = self.get_query()
+
+        # check if query has included_ids & if there are any ids in it
+        try:
+            if len(query['included_ids']) == 0:
+                raise KeyError
```

Comment:
Don't need a try/except block just to catch a failed "if" check. Instead you could just do something like:
```
if not query.get('included_ids'):
     .... Logic without included_ids ....
     return qs[:count]
.... 

Also, why the conditional behavior if no included_ids -- at least toss a comment in to explain.